src/main/resources/assets/warpdrive/lang/zh_cn.lang

```diff
@@ -686,20 +686,20 @@ warpdrive.command.lost_bed_can_t_teleport_other=玩家 %1$s 传送失败：\n床
 
 warpdrive.beam_frequency.tooltip=光束频率已设为%1$d。
 warpdrive.beam_frequency.get=已从 %1$s 获取光束频率 %2$d。
-warpdrive.beam_frequency.set=%1$s 已校准到频率 %2$d 。
+warpdrive.beam_frequency.set=%1$s 已校准到频率 %2$d。
 warpdrive.beam_frequency.status_line.valid=光束频率%1$d有效。
 warpdrive.beam_frequency.status_line.invalid=光束频率%1$d无效。
 warpdrive.beam_frequency.status_line.undefined=未定义的光束频率。\n§b使用一个音叉§r进行设置。
 
 warpdrive.control_channel.tooltip=控制频道设置为 %1$d。
-warpdrive.control_channel.get=已从 %1$s 获取控制频道 %2$d 。
-warpdrive.control_channel.set=%1$s 已调节到控制频道 %2$d 。
+warpdrive.control_channel.get=已从 %1$s 获取控制频道 %2$d。
+warpdrive.control_channel.set=%1$s 已调节到控制频道 %2$d。
 warpdrive.control_channel.status_line.valid=控制频道 %1$d 有效。
 warpdrive.control_channel.status_line.invalid=控制频道 %1$d 无效。
 warpdrive.control_channel.status_line.undefined=未定义控制频道。\n§b使用音叉§r进行设置。
 
-warpdrive.machine.is_enabled.set.enabled=开关打开！
-warpdrive.machine.is_enabled.set.disabled=开关关闭！
+warpdrive.machine.is_enabled.set.enabled=已启动！
```

Comment:
For reference, when the original message says "switched on" or "the machine is now enabled", we intend to say "the machine is now allowed to work".
Other conditions are required before we could say "the machine is now actually working".